### DISCUSSION TITLE
Comparisons to ignite

### DISCUSSION THREAD

**Discussion by techknowlogick** [weaveworks/flintlock#151] [Q&A] [answered]
Hi 👋 

Thanks for opensourcing this project. I was wondering since weaveworks already has `ignite` and that it seems to achieve similar goals to `reignite`, is it reasonable to assume that `reignite` is the successor to `ignite` (given this project's name)?

Thanks 🙂

**Comment by richardcase** [comment, marked as answer]
Hi @techknowlogick 

`flintlock` (previously known as `reignite`) achieves similar goals to `ignite` in that at their core both create Firecracker microvms and use OCI images.  ignites original use case was to bring a docker cli experience to Firecracker. However, `flintlock` is designed to be API driven (not cli drive) and to be used with a Cluster API provider we are building...that combined will result in Firecracker based Kubernetes clusters. So its more like the evolution of Ignite+Firekube with a focus on Cluster API. Also, we will be adding support for cloud hypervisor in addition to Firecracker.

Does that help?

**Reply by techknowlogick** [reply]
Awesome! Thanks for that detailed explanation:)
